client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2156,6 +2156,20 @@ Undocumented.prototype.transferStatus = function( siteId, transferId ) {
 };
 
 /**
+ * Get user suggestions for a site.
+ *
+ * @param {int}       siteId            The site ID
+ * @returns {Promise}
+ */
+Undocumented.prototype.getUserSuggestions = function( siteId ) {
+	debug( '/users/suggest/ query' );
+
+	return this.wpcom.users().suggest( {
```

Comment:
Should we be filtering on this endpoint by a search query? It seems strange as-is, as though the suggestions endpoint is basically the same as `/sites/%s/users` with pagination removed.